docs/design/proposals/configure-snapshot-based-rbd-mirroring.md

```diff
@@ -0,0 +1,324 @@
+# Configure Snapshot-based RBD Mirroring
+
+---
+RBD mirroring is an asynchronous replication of RBD images between multiple
+Ceph clusters. The following guide demonstrates how to perform the basic
+administrative tasks to configure
+[snapshot-based mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#create-image-mirror-snapshots)
+using the rbd command and handle Disaster Recovery scenarios.
+
+For more details about RBD Mirroring, refer to the official
+[Ceph documentation](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/).
+
+## Requirements
+
+* [Rook](https://rook.io/docs/rook/master/) should be installed
+  on the kubernetes cluster.
+  Refer [this](https://rook.io/docs/rook/master/ceph-quickstart.html) for rook installation.
+
+* Two Ceph clusters, such that, nodes on both clusters can connect to the nodes
+  in the other cluster.
+
+  > Note: snapshot-based mirroring requires the Ceph [Octopus](https://docs.ceph.com/en/latest/releases/)
+release or later.
+
+## Create Ceph Pools
+
+* Create a Ceph pool. For that, create `pool-test.yaml`
+
+   ```yaml
+    apiVersion: ceph.rook.io/v1
+    kind: CephBlockPool
+    metadata:
+      name: replicapool
+      namespace: rook-ceph
+    spec:
+      replicated:
+        size: 1
+      mirroring:
+        enabled: true
+        mode: image
+      statusCheck:
+        mirror:
+          disabled: false
+          interval: 60s
+    ```
+
+* Creating `replicapool` pool on primary cluster:
+
+  `[cluster-a]$ kubectl create -f pool-test.yaml --context=cluster-a`
```

Comment:
@pkalever this formatting seems to be accepted too. I like the `[hostname]$ command` format, as it looks pretty common.